pkg/volume/util/metrics.go

```diff
@@ -49,12 +49,12 @@ func registerMetrics() {
 }
 
 // OperationCompleteHook returns a hook to call when an operation is completed
-func OperationCompleteHook(plugin, operationName string) func(error) {
+func OperationCompleteHook(plugin, operationName string) func(*error) {
 	requestTime := time.Now()
-	opComplete := func(err error) {
+	opComplete := func(err *error) {
```

Comment:
Was there any particular reason to change this to address of an interface?

Reply:
@gnufied pass error as value, will cause the passed err to `operationcomplete` is always `nil`, and metric `storageOperationErrorMetric` doesn't work

Reply:
yep. @davidz627 explaind that below.